src/components/HotRepositories.tsx

```diff
@@ -1,138 +1,202 @@
+    }
   }
 
   return (
     <div className="flex flex-col px-4 max-w-screen-xl mx-auto">
       <div className="flex space-x-3 items-center">
-        <img src={hotIcon} alt="Hot Repo Icon" className="h-5 w-5" />
+        <img
+          src={hotIcon}
+          alt="Hot Repo Icon"
+          className="h-5 w-5"
+        />
+
         <h1 className="text-white font-bold text-2xl">Hot Repositories</h1>
       </div>
+
       <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4 w-full my-5">
         {hotRepos.map(
           ({ id, full_name, name, description, issues, stars, contributions }) => (
-            <div key={id} className="p-4 border rounded-lg bg-white w-full space-y-1 relative">
+            <div
+              key={id}
+              className="p-4 border rounded-lg bg-white w-full space-y-1 relative"
+            >
               {/* header & upvote button */}
+
               <div className="flex justify-between w-full">
                 <div className="flex space-x-1 items-center">
-                  <img src={getAvatarLink(full_name.replace(`/${name}`, ''))} alt="Hot Repo Icon" className="h-4 w-4 rounded-md overflow-hidden" />
-                  <span className="text-xs text-gray-400">{full_name.replace(`/${name}`, '')}</span>
+                  <img
+                    src={getAvatarLink(full_name.replace(`/${String(name)}`, ""))}
+                    alt="Hot Repo Icon"
+                    className="h-4 w-4 rounded-md overflow-hidden"
+                  />
+
+                  <span className="text-xs text-gray-400">{full_name.replace(`/${String(name)}`, "")}</span>
                 </div>
+
                 <button
                   className={`px-2 py-0.5 border rounded-lg flex justify-center items-center space-x-1 text-xs transition-all duration-200 ${
-                    checkVoted(id) ? "text-saucyRed border-saucyRed " : "text-grey border-gray-500 "
+                    checkVoted(id) && "text-saucyRed border-saucyRed " || "text-grey border-gray-500 "
                   }`}
-                  onClick={() => (user_id ? handleVoteUpdateByRepo(0, id) : signIn({ provider: "github" }))}
+                  onClick={async () => (user_id ? handleVoteUpdateByRepo(0, id) : signIn({ provider: "github" }))}
                 >
                   <span className="">{checkVoted(id) ? "voted" : "upvote"}</span>
-                  {checkVoted(id) ? <RiCheckboxCircleFill className="" /> : <FaArrowAltCircleUp className="" />}
+
+                  {checkVoted(id) && <RiCheckboxCircleFill className="" /> || <FaArrowAltCircleUp className="" />}
                 </button>
               </div>
+
               {/* repo name & description */}
+
               <div className="flex flex-col pb-10">
                 <a
                   href={`https://app.opensauced.pizza/repos/${full_name}`}
                   target="_blank"
-                  rel="noopener"
+                  rel="noopener noreferrer"
```

Comment:
this is not correct, I see the linter can fix _too much_ sometimes ...
@bdougie we should take a second look after final form is available, currently done 2/3 (prettier, react, typescript)

Reply:
interesting.

Reply:
Branched to beta, will leave this in for now! 🍕